lib/grizzly/zwave/command_classes.ex

```diff
@@ -105,7 +105,7 @@ defmodule Grizzly.ZWave.CommandClasses do
       {0x82, :hail},
       {0x84, :wake_up},
       {0x85, :association},
-      {0x86, :version},
+      {0x86, :command_class_version},
```

Comment:
The command class is `version` and it contains the command class get and command class report command.